cp-quickstart/start-docker-cloud.sh

```diff
@@ -5,95 +5,162 @@
 #########################################
 
+NAME=`basename "$0"`
 
 # Source library
-. ../utils/helper.sh
+source ../utils/helper.sh
+
+check_jq \
+  && print_pass "jq found"
 
 check_ccloud_version 1.0.0 \
-  && print_pass "ccloud version ok" \
-  || exit 1
+  && print_pass "ccloud version ok"
+
 check_ccloud_logged_in \
-  && print_pass "logged into ccloud CLI" \
-  || exit 1
-
-echo
-wget -O docker-compose.yml https://raw.githubusercontent.com/confluentinc/cp-all-in-one/${CONFLUENT_RELEASE_TAG_OR_BRANCH}/cp-all-in-one-cloud/docker-compose.yml
-
-echo ====== Confirm
-prompt_continue_cloud_demo || exit 1
-read -p "Do you acknowledge this script creates a Confluent Cloud KSQL app (hourly charges may apply)? [y/n] " -n 1 -r
-echo
-if [[ ! $REPLY =~ ^[Yy]$ ]]; then exit 1; fi
-
-echo
-echo ====== Create new Confluent Cloud stack
-cloud_create_demo_stack true || exit 1
-SERVICE_ACCOUNT_ID=$(ccloud kafka cluster list -o json | jq -r '.[0].name' | awk -F'-' '{print $4;}')
+  && print_pass "logged into ccloud CLI" 
+
+print_pass "Prerequisite check pass"
+
+[[ -z "$AUTO" ]] && {
+  printf "\n====== Confirm\n\n"
+  prompt_continue_cloud_demo || exit 1
+  read -p "Do you acknowledge this script creates a Confluent Cloud KSQL app (hourly charges may apply)? [y/n] " -n 1 -r
+  if [[ ! $REPLY =~ ^[Yy]$ ]]; then exit 1; fi
+	printf "\n"
+} 
+
+printf "\nFor your reference the demo will highlight some commands in "; print_code "code format"
+printf "\nFor example:\n"
+print_code_pass -c "successful command";
+printf "\n"
+print_code_error -c "failed command" -m "failed command message"
+
+printf "\n====== Starting\n\n"
+
+wget -q -O docker-compose.yml https://raw.githubusercontent.com/confluentinc/cp-all-in-one/${CONFLUENT_RELEASE_TAG_OR_BRANCH}/cp-all-in-one-cloud/docker-compose.yml \
+  && print_pass "retrieved docker-compose.yml from https://github.com/confluentinc/cp-all-in-one/cp-all-in-one-cloud/docker-compose.yml" \
+  || exit_with_error -c $? -n "$NAME" -m "could not obtain cp-all-in-one docker-compose.yml" -l $(($LINENO -2))
+
+printf "\n====== Creating new Confluent Cloud stack using the cloud_create_demo_stack function\nSee: %s for details\n" "https://github.com/confluentinc/examples/blob/$CONFLUENT_RELEASE_TAG_OR_BRANCH/utils/helper_cloud.sh"
+cloud_create_demo_stack true  \
+	&& print_code_pass -c "ccloud_create_demo_stack true"
+
+SERVICE_ACCOUNT_ID=$(ccloud kafka cluster list -o json | jq -r '.[0].name | split("-")[3]')
 CONFIG_FILE=stack-configs/java-service-account-$SERVICE_ACCOUNT_ID.config
 export CONFIG_FILE=$CONFIG_FILE
 check_ccloud_config $CONFIG_FILE || exit 1
-../ccloud/ccloud-generate-cp-configs.sh $CONFIG_FILE
-DELTA_CONFIGS_DIR=delta_configs
-source $DELTA_CONFIGS_DIR/env.delta
+
+../ccloud/ccloud-generate-cp-configs.sh $CONFIG_FILE \
+	&& print_code_pass -c "../ccloud/ccloud-generate-cp-configs.sh $CONFIG_FILE"
+
+DELTA_CONFIGS=delta_configs/env.delta
```

Comment:
Could `DELTA_CONFIGS` be renamed to `DELTA_CONFIGS_ENV`?  There are lots of delta configs for a variety of components...this demo is using the ones specifically for the env

Reply:
ack, updated